pyproject.toml

```diff
@@ -12,6 +12,7 @@ cairo-lang = "^0.10.1"
 openzeppelin-cairo-contracts = "0.5.0"
 amarna = "^0.1.5"
 starknet-py = "^0.14.0a0"
+pytest-asyncio = "^0.21.0"
```

Comment:
quite surprised to see this dep missing! However we only need to add it in dev.deps and not in both